include/maliput/api/road_geometry.h

```diff
@@ -209,6 +209,16 @@ class RoadGeometry {
   /// @return maliput's Inertial Frame to Backend Frame translation vector.
   math::Vector3 inertial_to_backend_frame_translation() const;
 
+  /// Takes a custom command formatted as a string and returns a string to be parsed to extract the output.
+  ///
+  /// @param command String containing the query for the backends to resolve, followed by the input paramters.
+  ///                 They should be formatted the following way:
+  ///                 `"<QueryForBackendsToResolve>,<param_1_name>:<param_1_value>,<param_2_name>:<param_2_value>,..."`.
```

Comment:
I would say that the backend should define its format, actually say that in the docs